quick_info/models.py

```diff
@@ -757,17 +773,18 @@ def retrieve_quick_info_master(self, quick_info_master_id, quick_info_master_we_
             'quick_info_master':        quick_info_master,
         }
         return results
-
-    def update_or_create_quick_info_master(self, quick_info_master_id,
-                                           quick_info_master_we_vote_id,
-                                           master_entry_name,
-                                           info_html, info_text,
-                                           language,
-                                           kind_of_ballot_item,
-                                           last_editor_we_vote_id,
-                                           more_info_url,
-                                           more_info_credit,
-                                           ):
+        
+    @staticmethod
+    def update_or_create_quick_info_master(
+		    quick_info_master_id,
+		    quick_info_master_we_vote_id,
+		    master_entry_name,
+		    info_html, info_text,
```

Comment:
8 character indention and info_html & info_text should each be on their own line.